styles.css

```diff
@@ -44,11 +47,24 @@ section {
   padding: 1em;
   text-decoration: none;
   transition: all 0.05s linear;
+  text-align: center;
 }
 
 .btn:hover {
   color: #f7ca4e;
   border-color: #f7ca4e;
   background-color: #000;
   border-radius: 5px;
-}
\ No newline at end of file
+}
+
+/* Media Query for mobile*/
+@media screen and (max-width: 335px) {
+  .main-img {
+    display: block;
+    margin-left: auto;
+    margin-right:auto;
+    width: 100%;
+  }
+  .main-title {
+    text-align: center;
+}
```

Comment:
@jkessler5535 there should be another closing `}` here to close your media query.